src/SEnergyCorrelator.ana.h

```diff
@@ -125,7 +138,45 @@ namespace SColdQcdCorrelatorAnalysis {
 
   }  // end 'DoLocalCalcWithPackage(double)'
 
-
+  void SEnergyCorrelator::DoLocalCalcManual(uint64_t jetIdx, vector<fastjet::PseudoJet> cstPseudoJet){
```

Comment:
Thanks for making this operate on a vector of `PseudoJet`s! This makes the code a _lot_ more general. However, the jet index still ties it to the local calculation. So I would suggest:

1. Instead replace the index with a double, what you'll normalize the weights with. In the case of the local calculation, this could be either the jet pT, eT, energy, or just 1. For the global calculation, you could then provide the relevant normalization (I think people frequently use the sum of jet pT in an event).
2. Then provide a default argument (done in the header file) and move the normalization to the end of the argument list: this helps communicate to the reader what's necessary for the function and gives a strong indication about what's going to be done.

```suggestion
  void SEnergyCorrelator::DoLocalCalcManual(const vector<fastjet::PseudoJet> cstPseudoJet, const double normalization) {
```